Bottom: dfec70d648229122a223c74fd6c162bb7057df14
Top:    1520c2d07424849e7f892c80248c4e19cec7a862
Author: sean.hefty@intel.com <sean.hefty@intel.com>
Date:   2012-07-11 12:39:24 -0700

Refresh of incl-com

---

diff --git a/Makefile.am b/Makefile.am
index b271b45..058b042 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -123,7 +123,7 @@ man_MANS = \
 	man/rdma_cm.7 \
 	man/rsocket.7
 
-EXTRA_DIST = src/cma.h src/indexer.h src/librdmacm.map \
+EXTRA_DIST = src/cma.h src/indexer.h src/librdmacm.map examples/common.h \
 	librdmacm.spec.in $(man_MANS)
 
 dist-hook: librdmacm.spec
